src/Project/Wooli/client/client.index.tsx

```diff
@@ -86,8 +87,18 @@ const initialize = async (jssState?: LayoutServiceData, dictionary?: i18n.Resour
   // if sitecoreData is not available fetch it from SC
   if (!jssState) {
     try {
-      const { language, itemPath } = getRoute(location.pathname);
-
+      const search = queryString.parse(location.search);
+      let language: string, itemPath: string;
+      //sc_itemid means we're in Experience Editor
+      if(search["sc_itemid"]) {
+        itemPath = search["sc_itemid"];
+        language = search["sc_lan"];
+      }
+      else {
+        const languageItemPath = getRoute(location.pathname);
```

Comment:
You can use destructing with already declared variable

Reply:
fixed